packages/connect/src/events/device.ts

```diff
@@ -9,7 +9,11 @@ import type {
 import type { Device } from '../types/device';
 import type { MessageFactoryFn } from '../types/utils';
 
-export { type DecodedTrezorPushNotification } from '@trezor/protocol';
+export {
+    type DecodedTrezorPushNotification,
+    TrezorPushNotificationMode,
+    TrezorPushNotificationType,
+} from '@trezor/protocol';
```

Comment:
I wouldn't do this. this probably adds some crypto dependencies via index reexport. use this pls https://github.com/trezor/trezor-suite/pull/21880/files#diff-52d3095a34f9e5924776bf02075ee5c7da25bb4d848d049139449cd9d21e8a54R12-R16

Reply:
True, changed a9941b9b7d5fc88f1a7b96fc72e66f91ee3ab700